packages/backend/src/utils/parsers/quadlet-unit-parser.ts

```diff
@@ -73,7 +125,7 @@ export class QuadletUnitParser extends Parser<string, Quadlet> {
       state: 'unknown',
       type: type,
       requires: unit.Requires,
-      resources: [], // TODO
+      resources: this.findResources(unit.SourcePath, raw),
```

Comment:
remark: this only look ups for files so the property and method could be renamed to files instead of resources